arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/dispatchers.kt

```diff
@@ -0,0 +1,36 @@
+package arrow.fx.reactor.extensions
+
+import reactor.core.scheduler.Scheduler
+import reactor.core.scheduler.Schedulers
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.ContinuationInterceptor
+import kotlin.coroutines.CoroutineContext
+
+internal val ComputationScheduler: CoroutineContext =
+  Schedulers.parallel().asCoroutineContext()
+
+internal val IOScheduler: CoroutineContext =
+  Schedulers.elastic().asCoroutineContext()
+
+fun Scheduler.asCoroutineContext(): CoroutineContext =
+  SchedulerContext(this)
+
+private class SchedulerContext(val scheduler: Scheduler) : AbstractCoroutineContextElement(ContinuationInterceptor), ContinuationInterceptor {
```

Comment:
I thought we had this already someplace else.

Reply:
For RxJava.